src/app/pagamentoplanos/service/pagamentoplanos.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { PagamentoplanosService } from './pagamentoplanos.service';
+
+describe('PagamentoplanosService', () => {
+  let service: PagamentoplanosService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(PagamentoplanosService);
+  });
+
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
  it("should be created", () => {
```